### PR TITLE
Feature/placeholder

### PR DESCRIPTION
**Pull request by LisaQ04**
F

### CODE REVIEW

File: waterlog/.env.development [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_API_URL=https://localhost:5001
\ No newline at end of file
+REACT_APP_API_URL=https://localhost:44383
```
Comment (FredJEhlers) [inline waterlog/.env.development:1]: dont commit `.env` files

File: waterlog/src/Components/MapFullScreen.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+import { generateMapIcons, levelToIntensity } from "../utils";
+
+const southWest = [-25.784510, 28.334360];
+const northEast = [-25.782110, 28.338325];
+const maxIntensity = 5;
+const centerPosition = [-25.783425, 28.336046];
+const defaultZoom = 17;
+
+function getHeatMapData({ monitorsCoordinates, segmentCoordinates }) {
+  let monitorMapData = monitorsCoordinates.map(mon => [mon.lat, mon.long, levelToIntensity(mon.faultLevel, maxIntensity)]);
```
Comment (FredJEhlers) [inline waterlog/src/Components/MapFullScreen.js:22]: split over two lines

File: waterlog/src/Components/SegmentHistory.js [hunk 3]
```diff
@@ -2,21 +2,21 @@ import React, { Component } from 'react';
...
-		this.state = {
-			id: this.props.match.params.id
-		}
-	}
-	render() {
-		return (
-			<div>
-				{}
-				<p className="home-text">Segment {this.state.id}</p>
+    constructor(props){
```
Comment (FredJEhlers) [inline waterlog/src/Components/SegmentHistory.js]: in the one file you are using 4 tabs eg `waterlog/src/Components/SegmentLeak.js` but in this file `waterlog/src/Components/SegmentHistory.js` you are using 2 tabs 

please use your project standard

File: waterlog/src/ConfigRedux/middleware/signalRMiddleware.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    HubConnectionBuilder,
+    LogLevel
+} from '@aspnet/signalr';
+
+import {
+    FETCH_ALERTS_BEGIN
+} from '../../actions/Types';
+
+const onNotifReceived = res => {
+    console.log('****** NOTIFICATION ******', res);
```
Comment (FredJEhlers) [inline waterlog/src/ConfigRedux/middleware/signalRMiddleware.js]: Do you need these logs

File: waterlog/src/Stylesheets/_map.scss [hunk 5]
```diff
@@ -21,20 +20,43 @@
...
+  max-width: 700px;
+  position: absolute;
+}
+
 .map-button-div-layer2 {
-	z-index: 2;
-	bottom: 0;
+  z-index: 2;
+  bottom: 0%;
+	margin: 9px auto;
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_map.scss:33]: indentation

Comment (FredJEhlers) [reply waterlog/src/Stylesheets/_map.scss:33]: line 39 as well

Comment (FredJEhlers) [reply waterlog/src/Stylesheets/_map.scss:33]: just check the whole file

File: waterlog/src/Stylesheets/_progressBar.scss [hunk 6]
```diff
@@ -5,44 +5,15 @@
   margin: 0 10px;
 }
 
-.filler-low {
-  background-color: #FFEA00 ;
+.filler {
+  background-color: red;
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_progressBar.scss:9]: use rgb like the other places have

File: waterlog/src/Stylesheets/_segmentLeak.scss [hunk 7]
```diff
@@ -1,53 +1,46 @@
...
-    position: relative;
 }
 
 .leakInfo {
     margin: 0 20px;
     padding: 0;
 }
 .low {
-    color: #FFEA00 !important;
+    color: #FFEA00;
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_segmentLeak.scss:10]: shame rgb comment

Comment (FredJEhlers) [reply waterlog/src/Stylesheets/_segmentLeak.scss:10]: Standardise your colour, rgb or hex but choose one for the project

File: waterlog/src/Stylesheets/_tank.scss [hunk 8]
```diff
@@ -21,42 +21,34 @@ h6{
...
     display: inline-block;
-    top: -44px;
-    right:39px;
-    color: black;
+    top: -60px;
+    color: #FFF;
     font-size: 10pt;
-    font-weight: bold;
-}
+}
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_tank.scss]: empty line EOF

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/.env.development (hunk 1 above)
Comment (LisaQ04) [reply waterlog/.env.development:1]: We are not storing any passwords, its just url.

File: waterlog/src/ConfigRedux/middleware/signalRMiddleware.js (hunk 4 above)
Comment (LisaQ04) [reply waterlog/src/ConfigRedux/middleware/signalRMiddleware.js]: File was no longer usable. Its been removed.

File: waterlog/src/Stylesheets/_tank.scss (hunk 8 above)
Comment (LisaQ04) [reply waterlog/src/Stylesheets/_tank.scss]: left empty line EOF.

File: waterlog/src/Stylesheets/_segmentLeak.scss (hunk 7 above)
Comment (LisaQ04) [reply waterlog/src/Stylesheets/_segmentLeak.scss:10]: Sure thanks.

File: waterlog/src/Stylesheets/_progressBar.scss (hunk 6 above)
Comment (LisaQ04) [reply waterlog/src/Stylesheets/_progressBar.scss:9]: Sure thanks, My group member is working on that.

File: waterlog/src/Components/MapFullScreen.js (hunk 2 above)
Comment (LisaQ04) [reply waterlog/src/Components/MapFullScreen.js:22]: Done

File: waterlog/src/Stylesheets/_map.scss (hunk 5 above)
Comment (LisaQ04) [reply waterlog/src/Stylesheets/_map.scss:33]: fixed.

File: waterlog/src/Components/SegmentHistory.js (hunk 3 above)
Comment (LisaQ04) [reply waterlog/src/Components/SegmentHistory.js]: Thanks fixed tabs

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**